build/BuildNuGets.csx

```diff
@@ -27,17 +28,30 @@ foreach (var file in files)
 {
     var nugetArgs = $@"pack {file} " +
         $@"-BasePath ""{binDir}"" " +
+        $@"-OutputDirectory ""{outDir}"" " +
         "-ExcludeEmptyDirectories " +
         $@"-prop licenseUrl=""{licenseUrl}"" " +
         $@"-prop version=""{buildVersion}"" " +
         $"-prop authors={authors} " +
         $@"-prop projectURL=""{projectURL}"" " +
         $@"-prop tags=""{tags}""";
-    Console.WriteLine(nugetArgs);
-    var p = Process.Start(Path.GetFullPath("../../nuget.exe"), nugetArgs);
+    var nugetExePath = Path.GetFullPath("../../nuget.exe");
```

Comment:
I found relative paths in this script a little confusing since I couldn't figure out the working directory just by looking at it. What about adding a `project-root` parameter and deriving both paths from it?